test/demo/utils/delay.ts

```diff
@@ -10,5 +10,6 @@ export async function delay(duration: number) {
 
 export enum Duration {
     PAGE_RELOAD = 500,
-    PAGE_NAVIGATE = 500
+    PAGE_NAVIGATE = 500,
+    LISTENER_TIMEOUT = 10000
```

Comment:
Is timeout the correct term here? This is just the long preregister delay, right? Would have both this and the short delay here, or just hardcode both where used

Reply:
Sure. I wasn't really sure what to call it and probably rushed a bit getting it in. I'll go back and have a look.

Reply:
Cleaned this up as discussed.